### ISSUE TITLE
Gradle error API 'variant.getMergeAssets()' is obsolete

### ISSUE THREAD

**Issue by fatimahossny** [csdcorp/speech_to_text#9]
I tried to build the project it gave me this errors
```
WARNING: API 'variant.getMergeAssets()' is obsolete and has been replaced with 'variant.getMergeAssetsProvider()'.
It will be removed at the end of 2019.
```
and this one too 
```
WARNING: API 'variantOutput.getProcessResources()' is obsolete and has been replaced with 'variantOutput.getProcessResourcesProvider()'.
It will be removed at the end of 2019.
```
Any help in this please ?

**Comment by sowens-csd** [comment]
So sorry, for some reason I haven't been getting notified on these issues. I haven't seen that message but I'll have a look now and update if required.

**Comment by sowens-csd** [comment]
I think this is the relevant Flutter issue. Looks like there's a fix either already done or imminent from them. https://github.com/flutter/flutter/issues/26544 I'll look into it further.

**Comment by sowens-csd** [comment]
This issue is resolved in 0.7.1 which upgrades the Android dependencies. Related to #17
